backend/app/routes/share.py

```
@@ -0,0 +1,167 @@
+        success=True, data=[_to_share(entry, port) for entry in share_registry_list()]
+    )
+
+
+# POST /share/albums/{album_id} - Start sharing an album over the network
+@router.post(
+    "/albums/{album_id}",
+    response_model=CreateShareResponse,
+    responses={code: {"model": ShareErrorResponseEnvelope} for code in [404, 500]},
+)
+async def create_share(
+    album_id: str = Path(...), body: CreateShareRequest = Body(default=None)
```

Ruff (0.16.1)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)